src/WebJobs.Script.Analyzers/CompilationWebJobsStartuptypeLocator.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using System.Reflection;
+using Microsoft.Azure.WebJobs.Hosting;
+
+namespace Microsoft.Azure.Functions.Analyzers
+{
+    /// <summary>
+    /// An implementation of an <see cref="IWebJobsStartupTypeLocator"/> that "locates" startup types
+    /// from a Microsoft.CodeAnalysis.Compilation's dependencies.
+    /// 
+    /// Note: Assemblies are derived from the Compilation by AssemblyCache. 
+    /// This class pulls the WebJobsStartupAttributes from assemblies in the list.
+    /// </summary>
+    public class CompilationWebJobsStartupTypeLocator : IWebJobsStartupTypeLocator
```

Comment:
Can delete this file as we do not need this file for the function name analyzer (current PR we want to merge). Add in the next PR where this will be actually used, as needed.

Reply:
Missed this; removing.